sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -77,11 +80,31 @@ protected void afterTest() {
         logger.info("Finished cleaning up values.");
     }
 
+    private ConfigurationAsyncClient getConfigurationAsyncClient(HttpClient httpClient,
```

Comment:
This one can replace beforeTest.

Reply:
Yes, this can replace it when all tests are updated to use parameters.